tests/apps/perf/pubsub_subscribe_http/app.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/websocket"
+)
+
+const (
+	pubSubName  = "kafka-messagebus"
+	topic       = "perf-test"
+	route       = "perf-test"
+	numMessages = 100
+)
+
+var upgrader = websocket.Upgrader{}
+
+// messagesCh contains the number of messages received
+var messagesCh = make(chan int, 100)
```

Comment:
Though it seems fine to have a hard-coded number here, as we are running perf test with certain assumptions here. But still, do you want to make it configurable, in case we would like to run perf test in future with different benchmarks with different number of messages and sizes?

Reply:
Nice suggestion, made it configurable via environment variables.